templates/user/user-about.html

```diff
@@ -45,6 +45,69 @@ <h4>{{ _('About') }}</h4>
             <br>
         {% endif %}
 
+        <h4>{{ _('Submission Activity') }}</h4>
```

Comment:
%d submissions in the last year?

Might as well switch the other headers on the user page to use sentence case, since most of the site uses sentence case.

Reply:
Done